final_task/README.md

```diff
@@ -1,3 +1,61 @@
-# Your readme here
-Some text.
-Checkout how to write this file using *markdown*.
+# Python RSS-reader 
+Python RSS-reader is a command-line utility which receives RSS URL and prints results in human-readable format.
+# Example:
+```shell
+$ python rss_reader.py "https://news.yahoo.com/rss/" --limit 1
+
+Feed: Yahoo News - Latest News & Headlines
+
+Title: Nestor heads into Georgia after tornados damage Florida
+Date: Sun, 20 Oct 2019 04:21:44 +0300
+Link: https://news.yahoo.com/wet-weekend-tropical-storm-warnings-131131925.html
+
+[image 2: Nestor heads into Georgia after tornados damage Florida][2]Nestor raced across Georgia as a post-tropical cyclone late Saturday, hours after the former tropical storm spawned a tornado that damaged
+homes and a school in central Florida while sparing areas of the Florida Panhandle devastated one year earlier by Hurricane Michael. The storm made landfall Saturday on St. Vincent Island, a nature preserve
+off Florida's northern Gulf Coast in a lightly populated area of the state, the National Hurricane Center said. Nestor was expected to bring 1 to 3 inches of rain to drought-stricken inland areas on its
+march across a swath of the U.S. Southeast.
+
+
+Links:
+[1]: https://news.yahoo.com/wet-weekend-tropical-storm-warnings-131131925.html
+[2]: http://l2.yimg.com/uu/api/res/1.2/Liyq2kH4HqlYHaS5BmZWpw--/YXBwaWQ9eXRhY2h5b247aD04Njt3PTEzMDs-/https://media.zenfs.com/en/ap.org/5ecc06358726cabef94585f99050f4f0
+
+```
+
+Utility has the following interface:
+```shell
+usage: rss_reader.py [-h] [--version] [--json] [--verbose] [--limit LIMIT]
+                     source
+
+Pure Python command-line RSS reader.
+
+positional arguments:
+  source         RSS URL
+
+optional arguments:
+  -h, --help     show this help message and exit
+  --version      Print version info
+  --json         Print result as JSON in stdout
+  --verbose      Outputs verbose status messages
+  --limit LIMIT  Limit news topics if this parameter provided
+
+```
+In case of using `--json` argument utility converts the news into [JSON](https://en.wikipedia.org/wiki/JSON) format.
+We can see JSON structure below:
+```
+{
+    "Feed": "Yahoo News - Latest News & Headlines",
+    "Title": "Nestor heads into Georgia after tornados damage Florida",
+    "Date": "Sun, 20 Oct 2019 04:21:44 +0300",
+  "Link": "https://news.yahoo.com/wet-weekend-tropical-storm-warnings-131131925.html",
+  "Links": [
+    "https://news.yahoo.com/wet-weekend-tropical-storm-warnings-131131925.html",
+    "http://l2.yimg.com/uu/api/res/1.2/Liyq2kH4HqlYHaS5BmZWpw--/YXBwaWQ9eXRhY2h5b247aD04Njt3PTEzMDs-/https://media.zenfs.com/en/ap.org/5ecc06358726cabef94585f99050f4f0"
+  ]
+}
+```
+The `--limit` argument also affects JSON generation.
+
+With the argument `--verbose` program prints all logs in stdout.
+
+Withe the argument `--version` program prints in stdout it's current version and complete it's work.
```

Comment:
You are supposed to write documentation yourself, Not to copy it from the task.

Reply:
Why? I did parser completely according to the task and all the options are listed in the task.

Reply:
But if you had inserted example of your program working you could have seen that there are some problems with encoding at least. We do not need to read about which arguments your application take because we know it anyway.  

We want you to describe how does your application work in the documentation and probably to list some peculiarities of your implementation, some explanations and etc.

Reply:
Got it